apps/skateboard/web/channels/game_channel.ex

```diff
@@ -16,24 +17,39 @@ defmodule Skateboard.GameChannel do
 
   def handle_in("board_state", payload, socket) do
     game = socket.assigns.game
-    push(socket, "board_state", Game.list_state(game))
+    push(socket, "board_state", Game.display_state(game))
 
     {:reply, {:ok, payload}, socket}
   end
 
   def handle_in("submit_word", letters, socket) do
-    game = socket.assigns.game
-    
-    # TODO - branch on success of word submission
-    Game.submit_word(game, letters, socket.assigns.user_id)
+    {validity, _} = Dictionary.check_word word(letters)
```

Comment:
Yeah, that crossed my mind too. At least this way you only need to pass back success from the Game, but it does seem a bit leaky, having game logic in the channel.